packages/scss/src/components/_table.scss

```diff
@@ -187,6 +187,15 @@
 			}
 		}
 	}
+
+	&.mod-stickyHeader {
+		.table-head-row-cell {
+			background-color: _color("white");
+			position: sticky;
+			top: 0;
```

Comment:
Le top va souvent être redéfinit mais par défaut je pense qu’il devrait être en fonction de la hauteur de la bannière.

Reply:
We also thought about this question. Maybe a solution would be to implement another specific mod with a `top: _theme("commons.banner-height")`